resources/dicts/relationship_events/normal_interactions/neutral.json

```diff
@@ -89,17 +84,14 @@
 		"id": "neutral_interaction4",
 		"interactions": [
 			"m_c stands tall when r_c walks by.",
-			"m_c doesn't mind r_c staring.",
 			"m_c tells r_c something that probably could've been kept private.",
-			"r_c sees m_c try and fail to something after claiming {PRONOUN/m_c/subject} {VERB/m_c/were/was} an expert at it.",
+			"r_c watches m_c try and fail at something after claiming {PRONOUN/m_c/subject} {VERB/m_c/were/was} an expert at it.",
 			"m_c puffs out {PRONOUN/m_c/poss} chest when r_c approaches.",
-			"m_c flashes a confident smile at r_c, but r_c doesn't know why.",

```

Comment:
Removed

Reply:
Removed since I didn't think it made much sense! It overlaps a little with the other neutral confident interactions ^^